x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/index.ts

```diff
@@ -5,4 +5,4 @@
  */
 export { AgentLogs } from './agent_logs';
 export { AgentDetailsActionMenu } from './actions_menu';
-export { AgentDetailsContent } from './agent_details';
+export { AgentDetailsContent } from './agent_details/';
```

Comment:
I believe the existing import will still work. Not a big deal either way, but I don't think it's is required or used elsewhere.